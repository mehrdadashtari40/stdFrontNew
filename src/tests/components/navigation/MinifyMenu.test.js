import React from 'react';
import { shallow } from 'enzyme';
import { MinifyMenu } from '../../../common';

test('Make Snapshot file , MinifyMenu Component', () => {
	const wrapper = shallow(<MinifyMenu />);

	expect(wrapper).toMatchSnapshot();
});
