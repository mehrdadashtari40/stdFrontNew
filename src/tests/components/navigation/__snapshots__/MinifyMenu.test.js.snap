// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make Snapshot file , MinifyMenu Component 1`] = `
<span
  className="minifyme"
  data-action="minifyMenu"
  onClick={[Function]}
>
  <i
    className="fa fa-arrow-circle-left hit"
  />
</span>
`;
